addons/l10n_it_edi/models/account_move.py

```diff
@@ -199,8 +199,32 @@ def action_l10n_it_edi_send(self):
         """
         self.ensure_one()
 
-        if self._l10n_it_edi_export_data_check():
-            raise UserError(_("The invoices you're trying to send have incomplete or incorrect data, please verify before sending."))
+        if errors := self._l10n_it_edi_export_data_check():
+            messages = []
+            for error_key, error_data in errors.items():
+                message = error_data['message']
+                split = error_key.split("_")
+                if len(split) > 3 and (model_id := {
+                    'partner': 'res.partner',
+                    'move': 'account.move',
+                    'company': 'res.company'
+                }.get(split[3], None)):
+                    if action := error_data.get('action'):
+                        if 'res_id' in action:
+                            record_ids = [action['res_id']]
+                        else:
+                            record_ids = action['domain'][0][2]
+                        records = self.env[model_id].browse(record_ids)
+                        message = f"{message} - {', '.join(records.mapped('display_name'))}"
+                messages.append(nl2br(message))
+
+            # Update the vendor bill's header with the warning messages,
+            # and force reload the view to make sure the header is loaded
+            self.l10n_it_edi_header = Markup('<br/>').join(messages)
```

Comment:
Hello @odoo/rd-security ! Can we get security exception for this one line of `Markup`?